### PR TITLE
Update EMDiffuse-n

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.10575472.
The following version(s) will be added:
  - [10575494 (version from 2024-01-27)](https://www.doi.org/10.5281/zenodo.10575494)
  - [10575473 (version from 2024-01-27)](https://www.doi.org/10.5281/zenodo.10575473)

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the bioimage.io CI tests: static (and dynamic) validations
- [ ] The meta information for the RDF item is complete
  - [ ] The tags are complete and describe the model
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D)
  - [ ] Authors are provided
  - [ ] Documentation is complete
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the bioimage.io admin team member.

Maintainers: @luchixiang

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: none

### CODE REVIEW

**Comment by Luchixiang** [comment]
Hi! How do I check whether the model is uploaded successfully or not? And can I modify the model name or tags or descriptions after uploading?

Thanks

**Comment by esgomezm** [comment]
Hi @Luchixiang 
Thank you for uploading the model. Before the model gets uploaded, it needs to pass different tests. You can see them below your comment (I add a screenshot). 
You can see that none of the two models passed the validation tests (dynamic validation), i.e., we cannot load your model and run it on the provided test images. This usually happens when something is missing in the model `rdf.yaml` file or the pytorch model is not exported properly. 
Here are some comments:
- Did you export the model using the `bioimageio.core` library? (https://github.com/bioimage-io/core-bioimage-io-python) You could try installing and using it to export the model, and validate the exported model. There are examples there, but let us know if it does not work. 
- I checked your models' config and it may be just that the pytorch version is missing in the `rdf.yaml` file. What's the pytorch version for this model? 
- Regarding the name and other information, you can upload a new version to the zenodo repo and update this information. A new PR like this one will be opened, tested and once everything is correct, we will merge it and the model will appear in the zoo. For now, I would suggest trying to get the model to pass the tests.

**Comment by Luchixiang** [comment]
Hi! Does the model output of test_input.npy needs to match test_output.npy in order to pass validation? The diffusion model is a generative model, so its output should be different each time.

**Comment by esgomezm** [comment]
Sorry, yes, the CI we use to test the models assumes that the output is always the same. This is something we are working on. Unfortunately, we do not have a solution for it yet. Sometimes, when fixing the seed of the diffusion model, it is possible to get the same output. Could you check if you could do so?
Also, it would be very helpful for us if you could get the model running properly and just the validation saying that the output is not the same (similar to this case https://github.com/bioimage-io/collection-bioimage-io/pull/686). This way, we know that we can at least, load and run your model, and we could develop further pipelines to test generative approaches.
